src/nodes/display/ToneMappingNode.js

```diff
@@ -11,30 +11,39 @@ import { NoToneMapping, LinearToneMapping, ReinhardToneMapping, CineonToneMappin
 // exposure only
 const LinearToneMappingNode = Fn( ( { color, exposure } ) => {
 
-	return color.mul( exposure ).clamp();
+	let c = vec3( color.rgb ).mul( exposure );
```

Comment:
@sunag

As I stated in my original post, `color` is RGBA in this case.

Can the changes I have made so far be implemented differently? Can the builder be modified to pass in an RGB color, instead?

How does the renderer differentiate between a color that is RGB vs RGBA?

Reply:
I have not tested this but you can try the following: In `setup()` change the line:
```js
const toneMappingParams = { exposure: this.exposureNode, color: colorNode };
```
to 
```js
const toneMappingParams = { exposure: this.exposureNode, color: colorNode.rgb };
```

And then change the following line from
```js
outputNode = toneMappingNode( toneMappingParams );
```
to 
```js
outputNode = vec4( toneMappingNode( toneMappingParams ), colorNode.a );
```

Reply:
It works! Thanks, @Mugen87!